addons/website_sale/views/snippets/s_products_searchbar.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_products_searchbar_input" name="Products Search Input">
+    <t t-call="website_sale.website_sale_products_search_box">
+        <t t-set="_classes" t-value="'s_wsale_products_searchbar_input mb-3'"/>
+    </t>
+</template>
+<template id="s_products_searchbar" name="Products Search">
+    <section class="s_wsale_products_searchbar bg-200 pt24 pb24">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 offset-lg-2">
+                    <div class="card">
+                        <div class="card-body">
+                            <h2>Search for a product</h2>
+                            <p>We have amazing products in our shop, check them now !</p>
+                            <t t-call="website_sale.s_products_searchbar_input"/>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+
+<template id="searchbar_input_snippet_options" inherit_id="website_sale.snippet_options" name="search bar snippet options">
+    <xpath expr="." position="inside">
+        <div data-selector=".s_wsale_products_searchbar_input" data-js="ProductsSearchBar">
+            <we-select string="Search filter to use by default" id="order_by" data-attribute-name="value" data-apply-to=".o_wsale_search_order_by">
+                <we-button data-select-attribute="">Default order</we-button>
+                <we-button data-select-attribute="name asc">Order by name (A-Z)</we-button>
+                <we-button data-select-attribute="name desc">Order by name (Z-A)</we-button>
+                <we-button data-select-attribute="list_price asc">Order by price: low to high</we-button>
+                <we-button data-select-attribute="list_price desc">Order by price: high to low</we-button>
+            </we-select>
```

Comment:
Options:
"
default (but what is it?)
by name (A-Z)
by name (Z-A)
by price (low to high)
by price (high to low)

Reply:
I suppose it is the default order of the search which is defined on the product mode:
`_order = 'complete_name'`

Reply:
So A-Z?

Reply:
I edited my message, It is by product code, then name, then ID

Reply:
@ThanhDodeurOdoo You are looking at the order of product.product which is not the right model but we don't even use the default search order anyway.

@sbuhl The default is the default order you chose for the shop page. `website_sequence, id`